vita3k/gui/src/settings_dialog.cpp

```diff
@@ -998,6 +998,12 @@ void draw_settings_dialog(GuiState &gui, EmuEnvState &emuenv) {
         if (ImGui::IsItemHovered())
             ImGui::SetTooltip("Dump loaded code as ELFs");
         ImGui::Spacing();
+        if (emuenv.backend_renderer == renderer::Backend::Vulkan) {
+            ImGui::Checkbox("Validation Layer (Reboot required for apply)", &emuenv.cfg.validation_layer);
```

Comment:
```suggestion
            ImGui::Checkbox("Validation Layer (Reboot required)", &emuenv.cfg.validation_layer);
```
"For apply" is not proper english

Reply:
done